types/balanced-match/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "type": "module",
+    "exports": {
+        ".": {
+            "types": {
+                "default": "./index.d.ts"
```

Comment:
Would `"types": "./index.d.ts"` achieve the same thing?

Unfortunately, one can't just copy/paste the upstream value since they wrote `"export": "./index.js"`, but I think what you have works fine.

Reply:
noted for future, thx!